File: Estudos/Dart/Sintaxe/Objeto.dart
void main() {

  Pessoa pessoa1 = new Pessoa();

  pessoa1.nome = 'Derek';
  pessoa1.idade = 17;
  pessoa1.telefone = '119876543210';

  pessoa1.apresenta();

}

class Pessoa {

  //Propriedades da Classe - Variaveis
  String nome = '';
  int idade = 0;
  String telefone = '';

  //Métodos da Classe - Funções
  void apresenta(){
    print( 'Meu nome é ${this.nome} e tenho ${this.idade} anos' );
  }
}

File: Estudos/TypeScript/Sintaxe/Objeto.ts
    /*
    let usuario: {
        nome: String,
        idade: number
    } = {
        nome: "Rafael",
        idade: 16
    }
    */

    let pessoa: object = {
        nome: "Rafael",
        idade: 16
    };

    interface usuario {
        nome: String;
        idade: number;
    }

    let user: usuario = {
        nome: "Rafael",
        idade: 16
    };

    console.log(user.nome);

    console.log(Object.keys(user)); // ["nome","idade"]

    console.log(Object.values(user)); // ["Rafael",20]

    console.log(Object.entries(user)); // [["nome","Rafael"],["idade",20]]
